src/components/icons/crud.tsx

```diff
@@ -1,71 +1,41 @@
-        fill={fillColor}
-      />
+    <svg viewBox="0 0 14 13" xmlns="http://www.w3.org/2000/svg" {...props}>
+      <path d="M12.3353 6.5479C12.1806 6.5479 12.0322 6.60936 11.9228 6.71875C11.8134 6.82815 11.752 6.97652 11.752 7.13123V10.6312C11.752 10.7859 11.6905 10.9343 11.5811 11.0437C11.4717 11.1531 11.3233 11.2146 11.1686 11.2146H3.00195C2.84724 11.2146 2.69887 11.1531 2.58947 11.0437C2.48008 10.9343 2.41862 10.7859 2.41862 10.6312V2.46456C2.41862 2.30985 2.48008 2.16148 2.58947 2.05209C2.69887 1.94269 2.84724 1.88123 3.00195 1.88123H6.50195C6.65666 1.88123 6.80504 1.81977 6.91443 1.71038C7.02383 1.60098 7.08529 1.45261 7.08529 1.2979C7.08529 1.14319 7.02383 0.994815 6.91443 0.885418C6.80504 0.776022 6.65666 0.714564 6.50195 0.714564H3.00195C2.53782 0.714564 2.0927 0.898939 1.76452 1.22713C1.43633 1.55532 1.25195 2.00044 1.25195 2.46456V10.6312C1.25195 11.0954 1.43633 11.5405 1.76452 11.8687C2.0927 12.1969 2.53782 12.3812 3.00195 12.3812H11.1686C11.6327 12.3812 12.0779 12.1969 12.4061 11.8687C12.7342 11.5405 12.9186 11.0954 12.9186 10.6312V7.13123C12.9186 6.97652 12.8572 6.82815 12.7478 6.71875C12.6384 6.60936 12.49 6.5479 12.3353 6.5479ZM3.58529 6.99123V9.46456C3.58529 9.61927 3.64674 9.76765 3.75614 9.87704C3.86554 9.98644 4.01391 10.0479 4.16862 10.0479H6.64195C6.71872 10.0483 6.79483 10.0336 6.8659 10.0046C6.93697 9.97557 7.00161 9.93279 7.05612 9.87873L11.0928 5.83623L12.7495 4.21456C12.8041 4.16034 12.8475 4.09582 12.8771 4.02473C12.9068 3.95365 12.922 3.8774 12.922 3.8004C12.922 3.72339 12.9068 3.64715 12.8771 3.57606C12.8475 3.50498 12.8041 3.44046 12.7495 3.38623L10.2761 0.883731C10.2219 0.829056 10.1574 0.785659 10.0863 0.756044C10.0152 0.726429 9.93896 0.711182 9.86195 0.711182C9.78495 0.711182 9.7087 0.726429 9.63762 0.756044C9.56653 0.785659 9.50201 0.829056 9.44779 0.883731L7.80279 2.53456L3.75445 6.57706C3.70039 6.63157 3.65762 6.69621 3.62859 6.76729C3.59956 6.83836 3.58484 6.91446 3.58529 6.99123ZM9.86195 2.1204L11.5128 3.77123L10.6845 4.59956L9.03362 2.94873L9.86195 2.1204ZM4.75195 7.2304L8.21112 3.77123L9.86195 5.42206L6.40279 8.88123H4.75195V7.2304Z" />
     </svg>
   );
 };
 
 export const Delete = (props: any) => {
-  const fillColor = "#372D56";
```

Comment:
Removing this programmatic fill setting in favor of using tailwind `fill-color` classes. They are a bit flakey but I got them to work with some wrappers.